### DISCUSSION TITLE
Formatar data REST API

### DISCUSSION THREAD

**Discussion by GuilhermeDeLucca** [LiveHelperChat/livehelperchat#1803] [Q&A]
Good evening, how can I format the API return date?

`[
    {
        "_id": "WybROSei3",
        "type": "text",
        "content": {
            "text": "Contratos Ativos\n\n{foreach=content_1}\nData Vencimento: {args.item.data_vencimento} Valor {args.item.valor_aberto}\n{/foreach}"
        }
    }
]`

my return

Data Vencimento: 2023-01-15 Valor 99.90
Data Vencimento: 2022-12-15 Valor 99.90
Data Vencimento: 2022-11-15 Valor 99.90

how would you like

Data Vencimento: 15/01/2023 Valor 99.90
Data Vencimento: 15/12/2022 Valor 99.90
Data Vencimento: 15/11/2022 Valor 99.90

would it be with javascript? but how do i do that inside the foreach

`var dataInput = '{args.item.data_vencimento} ';

data = new Date(dataInput);
dataFormatada = data.toLocaleDateString('pt-BR', {timeZone: 'UTC'});`

**Comment by remdex** [comment]
Your question does not make sense, because code sample is for Rest API, and you asking about javacsript. Impossible to understand what you are trying to achieve.

**Reply by GuilhermeDeLucca** [reply]
Sorry if I couldn't be clear!

I will ask in a simpler way

Is it possible to format the return of a date from the REST API?

Which I get through (PS I love this implementation on the LHC):

```
{foreach=content_1}
- {args.item.payment_date}
{/foreach}
```

payment_date comes in the format yyyy-mm-dd

I want it in dd/mm/yyyy format

Is it possible to format this date in the LHC or should it already come formed from the API?

**Reply by remdex** [reply]
It should come from Rest API
